packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -41,6 +42,29 @@ export function buildRootObject(vatPowers) {
         repeaters.set(index, vatRepeater);
         return vatRepeater;
       },
+      createNotifier(delaySecs, interval) {
+        assert(
+          Nat(delaySecs) >= 0,
+          details`createNotifier's first parameter must be a non-negative integer. ${delaySecs}`,
+        );
+        assert(
+          Nat(interval),
+          details`createNotifier's second parameter must be an integer, ${interval}`,
+        );
```

Comment:
The test and the error text disagree. I see the same issue on line 26 above. A negative integer will be rejected with a complaint that it is not an integer.